lib/seattleflu/db/cli/command/reportable_conditions.py

```diff
@@ -0,0 +1,238 @@
+"""
+Run reportable condition notification routine.
+
+Sends a nicely formatted Slack message for samples with reportable conditions as
+described in the `warehouse.organism` table.
+
+Searches for unseen reportable conditions in `shipping.reportable_condition_v1`,
+and updates `warehouse.presence_absence` after sending a Slack alert.
+
+Visit the Reportable Conditions Notifications Slack app
+<https://api.slack.com/apps/ALJJAQGKH> to access the Slack Incoming Webhook
+URLs <https://api.slack.com/apps/ALJJAQGKH/incoming-webhooks?>. These URLs must
+be defined as environment variables outside of this command.
+
```

Comment:
This nice formatting below won't be kept because click reformats help text. If you run with `--help` you'll see the ugly results. You can fix this by adding a `\b` here. See examples of this elsewhere in ID3C.